.github/workflows/debos.yml

```diff
@@ -35,6 +35,12 @@ jobs:
         - /efs/qli/metaqcom/gh-runners/quic-yocto/downloads:/fileserver-downloads
       options: --privileged
     steps:
+      - name: TMP-DEBUG
+        uses: qualcomm-linux/upload-private-artifact-action@v1
+        with:
+          path: /tmp/${BUILD_ID}
+          build_id: "${{ github.repository }}-${{ github.run_id }}-${{ github.run_attempt }}"
+          github_token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
I'm surprised you need to pass this specifically in the yaml; isn't GITHUB_TOKEN provided to actions by default?

Reply:
Its a security thing. GitHub doesn't want you to accidentally share the token with a workflow you haven't explicitly granted permissions to.

Reply:
Is is possible to implement an action like the github upload-artifact action, without passing the token? Does that require some lengthy certification process with github or something?